tests/Resources/Page/ODH/AiApps/Openvino.robot

```diff
@@ -24,4 +24,6 @@ Verify JupyterHub Can Spawn Openvino Notebook
     Wait Until Element Is Enabled    xpath://input[@name="OpenVINO™ Toolkit"]    timeout=10
```

Comment:
could you add the Polarion ID to this test case?

Reply:
yes I have added it n the test where this keyword is used